docs/examples/CreatableAdvanced.tsx

```diff
@@ -25,50 +18,30 @@ const defaultOptions = [
   createOption('Three'),
 ];
 
-export default class CreatableAdvanced extends Component<{}, State> {
-  state: State = {
-    isLoading: false,
-    options: defaultOptions,
-    value: undefined,
-  };
-  handleChange = (
-    newValue: OnChangeValue<Option, false>,
-    actionMeta: ActionMeta<Option>
-  ) => {
-    console.group('Value Changed');
-    console.log(newValue);
-    console.log(`action: ${actionMeta.action}`);
-    console.groupEnd();
```

Comment:
I've removed the console logs to simplify the example. I don't think they were necessary for the purpose of this example.